Formula/i/ipopt.rb

```diff
@@ -18,7 +19,7 @@ class Ipopt < Formula
 
   depends_on "openjdk" => :build
   depends_on "pkg-config" => [:build, :test]
-  depends_on "ampl-mp"
+  depends_on "ampl-mp@3"
```

Comment:
When can we switch back? Is there an issue upstream to update?

Reply:
asl got removed in mp 4, so have to use mp3 for now, https://github.com/coin-or/Ipopt/issues/802

Reply:
Does https://github.com/ampl/asl/releases/tag/v1.0.0 work instead?

Reply:
yeah, it works fine for ipopt, so swap to use ampl-asl instead.